notebooks/README.md

```diff
@@ -1,22 +1,8 @@
-
-## ✅ Description
-This is a STT which was made as an assignment for my Programming course from MSc Voice Technology at Rijksuniversiteit Groningen/Campus Fryslân.
-
-The data is my 105 English recordings in [Common Voice](https://commonvoice.mozilla.org/en) (takeout_442_metadata.txt and takeout_442_pt_0.zip).
-
-
-## ✅ What did I do?
-How I first did it was just like the notebook but then hit the stone and realized the link will be expired after some time so I just downloaded my data and uploaded it here so I can train my STT easier.
+# Python Notebooks for 🐸 STT
 
 | Notebook title | Language(s) | Link to Colab |
```

Comment:
Specify that these are from the original and not yours (just to make things more explicit)